src/InputWithLabel.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+const InputWithLabel = ({
+    id,
+    todoTitle,
+    type='text',
+    handleTitleChange,
+    isFocused,
+    children,
+}) => {
+    const inputRef = React.useRef();
+
+    React.useEffect(() => {
+           if (isFocused && inputRef.current){
+               inputRef.current.focus();
+           }
+    }, [isFocused]);
+
+    return (
+        <>
+            <label htmlFor={id}>{children}</label>
+            <input
+                name="title"
+                type={type}
+                id={id}
+                ref={inputRef}
+                value={todoTitle}
+                onChange={handleTitleChange}>
+            </input>
```

Comment:
awesome sauce!! 🎖️ 💯 That's a nice component there. You can even make the `name` attribute a prop to make the component more reusable too.